docs/csaf_downloader.md

```diff
@@ -12,6 +12,7 @@ Application Options:
       --version                Display version of the binary
   -v, --verbose                Verbose output
   -r, --rate=                  The average upper limit of https operations per second
+  -w, --worker=                Number of concurrent downloads (default: 1)
```

Comment:
Maybe add that using not positive or too large numbers will try to use the maximum amount of workers avaible. E.g. 

    If worker is set to 0 or less, the maximum amount of available workers will be used.

Down there with the two other explanatory lines?

Reply:
I gave that a second thought and I have removed the CPU stuff entirely.
In commit 3da3b4877889e20b7d691c8792fd70e0a186db01 the lower bound is now 1.